src/main/resources/swagger/api-docs.yaml

```diff
@@ -2009,6 +2774,17 @@ components:
         policy:
           $ref: '#/components/schemas/AccessPolicyMembership'
       description: ""
+    AccessPolicyResponseEntryV2:
```

Comment:
I opted to make this a separate component instead of just adding descendant permissions as an optional property so that they wouldn't show up at all in the v1 payloads in swagger in an effort to be conscious of the swagger generated client by not changing the v1 endpoints at all